MapboxCoreNavigationTests/OfflineRoutingTests.swift

```diff
@@ -1,130 +0,0 @@
-import XCTest
-import MapboxDirections
-import TestHelper
-@testable import MapboxCoreNavigation
-
-class OfflineRoutingTests: XCTestCase {
-    func testOfflineDirections() {
-        let bundle = Bundle(for: Fixture.self)
-        let tilesURL = URL(fileURLWithPath: bundle.bundlePath.appending("/tiles/liechtenstein"))
```

Comment:
#2546 proposes reusing this test fixture, if not the overall test, to facilitate automated integration testing of the PassiveLocationDataSource in free driving mode.